apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/IterableContainsInOrderOnlyEntriesAssertionsSpec.kt

```diff
@@ -23,6 +24,14 @@ class IterableContainsInOrderOnlyEntriesAssertionsSpec : Spek({
         "[Atrium][Builder] "
     )
 
+    object OldShortcutSpec : ch.tutteli.atrium.spec.integration.IterableContainsInOrderOnlyEntriesAssertionsSpec(
```

Comment:
remove

Reply:
Done!

Reply:
It's still here :) did you push the wrong commit?